src/frontend/use-user.tsx

```diff
@@ -15,6 +15,7 @@ export interface UserProfile {
   picture?: string | null;
   sub?: string | null;
   updated_at?: string | null;
+  org_id?: string;
```

Comment:
```suggestion
  org_id?: string | null;
```

Reply:
I wouldn't expect `org_id` to ever be null (either `undefined` or a `string`) (so does spa-js https://github.com/auth0/auth0-spa-js/blob/d0578f63813e6b5d493b8bd92aea3ecfcde9fcb3/src/global.ts#L523)

But it does look weird having one without the `| null` so I'll put add it in

(also not sure why the tests are failing will take a look)